### ISSUE TITLE
Conflict with other JS plugin

### ISSUE THREAD

**Issue by nicolasbolzinger** [luruke/barba.js#337]
First, Merry Christmas and Happy New Year to everybody !

I'm using a keepTheRhythm.js plugin which automatically add padding to pictures in order to have a nice vertical rhythm. Thanks to this JS, the image height (+ padding) will be a multiple of my baseline.

I installed Barba, it works nice but i have to manually refresh the page because keepTheRhythm.js is not adding (automatically) any padding when i use barba.js.

I'm wondering if there is a way to run these 2 .js together ? Is there an order of .js loading to respect ? Do i have to load keepTheRhythm inside Barba ? 

Sorry for my english btw.

Here is my keepTheRhythm config file
```js
$(function() {
    $('figure').keepTheRhythm({
        baseLine: 12,
        verticalAlignment: 'top'
    });
});
$(function() {
    $('.post-thumbnail').keepTheRhythm({
        baseLine: 12,
        verticalAlignment: 'center'
    });
});
```

Here is the keepTheRhythm js file ([on github](https://github.com/Gaya/jQuery--Keep-the-Rhythm))
```js
;(function ( $, window, document, undefined ) {

    var pluginName = "keepTheRhythm",
        defaults = {
            baseLine: 24,
            verticalAlignment: "center",
            spacing: "padding"
        };

    // The actual plugin constructor
    function Plugin( element, options ) {
        this.element = element;

        // jQuery has an extend method which merges the contents of two or
        // more objects, storing the result in the first object. The first object
        // is generally empty as we don't want to alter the default options for
        // future instances of the plugin
        this.options = $.extend( {}, defaults, options );

        this._defaults = defaults;
        this._name = pluginName;

        this.init();
    }

    Plugin.prototype = {

        init: function() {
            var obj = $(this.element);
            var rhythmPlugin = this;

            $(window).resize(function () {
                rhythmPlugin.fixRhythm(obj);
            }).trigger("resize");
        },

        fixRhythm: function (obj) {
            var h = obj.height();
            var r = this.options.baseLine - (h % this.options.baseLine);

            //if the element is in rhythm: do nothing.
            if (r == this.options.baseLine) {
                r = 0;
            }

            var top = 0;
            var bottom = r;

            if (this.options.verticalAlignment == "center") {
                //if verticalAlignment is set to center; spread the padding to both top and bottom
                top = r / 2;
                bottom = r - top;
            } else if (this.options.verticalAlignment == "bottom") {
                //if the alignment is bottom; set to padding to the top!
                top = r;
                bottom = 0;
            }

            if (this.options.spacing == "margin") {
                obj.css({
                    marginTop: top + "px",
                    marginBottom: bottom + "px"
                });
            } else {
                obj.css({
                    paddingTop: top + "px",
                    paddingBottom: bottom + "px"
                });
            }
        }
    };

    $.fn[pluginName] = function ( options ) {
        return this.each(function () {
            if (!$.data(this, "plugin_" + pluginName)) {
                $.data(this, "plugin_" + pluginName, new Plugin( this, options ));
            }
        });
    };

})( jQuery, window, document );
```

My Barba config file
```js
$('document').ready(function(){
    Barba.Pjax.start();
});
```

**Comment by c0mrx** [comment]
Hi @nicolasbolzinger,

You'll need to use barba's events to rerun your JS scripts. Since the pages are pulled in via AJAX, the code isn't being rerun again. Check out issue #291.

**Comment by nicolasbolzinger** [comment]
It works !! Thank you !
